open_src/influx/coordinator/statement_executor.go

```diff
@@ -145,6 +145,11 @@ func (e *StatementExecutor) ExecuteStatement(stmt influxql.Statement, ctx *query
 			messages = append(messages, query.ReadOnlyWarning(stmt.String()))
 		}
 		err = e.executeCreateRetentionPolicyStatement(stmt)
+	case *influxql.CreateContinuousQueryStatement:
+		if ctx.ReadOnly { // Benevor TODO: for what?
```

Comment:
If it is ReadOnly, it usually means that there is not enough disk space.

Reply:
I understand.